db/migrate/20150821100137_migrate_from_scaptimony.rb

```diff
@@ -0,0 +1,61 @@
+class MigrateFromScaptimony < ActiveRecord::Migration
+  def up
+    ActiveRecord::Base.connection.tables.grep(/^scaptimony/).each do |table|
+      rename_table table, table.sub(/^scaptimony/, "foreman_openscap")
+    end
+
+    execute 'DROP VIEW scaptimony_arf_report_breakdowns' if table_exists? 'scaptimony_arf_report_breakdowns'
+    execute 'DROP VIEW foreman_openscap_arf_report_breakdowns' if table_exists? 'foreman_openscap_arf_report_breakdowns'
+
+    execute <<-SQL
+      CREATE VIEW foreman_openscap_arf_report_breakdowns AS
+        SELECT
+          arf.id as arf_report_id,
+          COUNT(CASE WHEN result.name IN ('pass','fixed') THEN 1 ELSE null END) as passed,
+          COUNT(CASE result.name WHEN 'fail' THEN 1 ELSE null END) as failed,
+          COUNT(CASE WHEN result.name NOT IN ('pass', 'fixed', 'fail', 'notselected', 'notapplicable') THEN 1 ELSE null END) as othered
+        FROM
+          foreman_openscap_arf_reports arf
+        LEFT OUTER JOIN
+          foreman_openscap_xccdf_rule_results rule
+          ON arf.id = rule.arf_report_id
+        LEFT OUTER JOIN foreman_openscap_xccdf_results result
+          ON rule.xccdf_result_id = result.id
+        GROUP BY arf.id;
+          SQL
+
+    taxonomies = TaxableTaxonomy.where(:taxable_type => ["Scaptimony::ArfReport", "Scaptimony::Policy", "Scaptimony::ScapContent"])
+    taxonomies.map { |t| t.taxable_type.sub!(/^Scaptimony/, "ForemanOpenscap")}
```

Comment:
this does not work, you map the taxonomies but throw the result away, either use `map!` or `taxonomies = taxonomies.map {...}`, similarilly in down